src/router/index.js

```diff
@@ -87,6 +88,11 @@ const routes = [
     meta: {
       requiresAuth: true
     }
+  },
+  {
+    path: "/personvern",
```

Comment:
Decided to go for a Norwegian URL, as the page is in that language. But this can of course be changed if we want all URLs to be in English.

Reply:
URLene våre er på engelsk. Norsk språk på github og engelske URLer på norsk side, it's a mad world I know.